### PR TITLE
grid

### PR DESCRIPTION
**Pull request by arpit01923**
A grid component is created in which-

1. 2-grid: 2 items in grid is created
2. 3-grid: 3 items in grid is created
image

### CODE REVIEW

**Review by aniketsingh13 — APPROVED** [review]
good to go

File: css/components/Grid.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    grid-column-gap: 1rem;
+}
+
+.grid-2{
+    grid-template-columns: 10rem 10rem;
+}
+
+.grid-3{
+    grid-template-columns: 10rem 10rem 10rem;
+}
```
Comment (laxmikanthkonthum) [inline css/components/Grid.css]: Suggestion: repeat(2,10rem) and repeat(3,10rem) can be used

**Review by laxmikanthkonthum — APPROVED** [review]
Looks good

File: css/components/Grid.css (hunk 1 above)
Comment (arpit01923) [reply css/components/Grid.css]: done

**Pull request merged**